cvat/apps/dataset_repo/dataset_repo.py

```diff
@@ -94,9 +94,9 @@ def _parse_url(self):
                 host = http_match.group(1)
                 repos = http_match.group(2)[1:]
             elif ssh_match:
-                user = ssh_match.group(1)
-                host = ssh_match.group(2)
-                repos = "{}{}".format(ssh_match.group(3), ssh_match.group(4))
+                user = ssh_match.group(1) + ssh_match.group(2) if ssh_match.groups(1) is not None else ssh_match.group(2)
```

Comment:
@matthias-p , I believe you have a typo here: `ssh_match.groups`

Reply:
@nmanovic , indeed, thank you, I fixed it.